recognition-server/server.py

```diff
@@ -14,7 +18,11 @@ def hello():
     if 'image' in request.files:
         f = request.files['image']
         ext = '.' + f.filename.split('.')[-1]
+        filePath = 'tmp/' + str(uuid.uuid4()) + ext
         f.save('tmp/' + str(uuid.uuid4()) + ext)
+
+        os.system('python Recognition.py')
+
         # Add here image processing
         # set payload
         return jsonify({'error' : 'false', 'message' : 'No vehicle ID found. Please retry or enter manually !', 'payload' : '123456'})
```

Comment:
as you now have **vehicle_id** as in above
you can return the real thing

if it is -1, then error is true, payload is -1 and message is same as in here

if its not -1 then error is false, payload is the number you have, and a meaningful message

Reply:
Add this in the bottom

```
if __name__ == '__main__':
    app.run(debug=True)
```